app/models/candidates/session_token.rb

```diff
@@ -31,10 +40,23 @@ def expire!
   end
 
   def invalidate_other_tokens!
-    candidate.session_tokens.unexpired.update_all(expired_at: Time.zone.now)
+    candidate.session_tokens.
+      unconfirmed.unexpired.
+      update_all(expired_at: Time.zone.now)
   end
 
   def to_param
     token
   end
+
+  def confirm!
+    raise CannotConfirmInvalidToken unless valid?
+
+    update!(confirmed_at: Time.zone.now)
+    invalidate_other_tokens!
```

Comment:
Do we want to invalidate other confirmed unexpired tokens here? Not sure on the CRM's business rules

Reply:
No we don't - /goes off to check behaviour/

Reply:
Once they are confirmed they are determined to be used anyway, we may want to limit users to a single sign in at some point but don't currently.

the `valid` scope is restricted to those which have neither been used (confirmed), nor expired

Based on the above I'll merge